src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -708,6 +708,26 @@ public synchronized StringBuffer reverse() {
         return this;
     }
 
+    /**
+     * @since 21
+     * @throws IllegalArgumentException {@inheritDoc}
```

Comment:
\@throws comes before \@since here and in StringBuilder.

Reply:
Changed